dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -246,3 +246,19 @@ properties:
       - 5
       - 6
       - 7
+
+  wakeup-duration:
+    type: int
+    default: 0
```

Comment:
Need to justify the default in the description. Is it a register reset value?

Reply:
Default value is a register reset value. Add this information to the description ?

Reply:
Yes, correct. Please add it in the description ("default value is the register reset value")